github/ci/prow/tasks/main.yml

```diff
@@ -138,6 +138,22 @@
           type: Opaque
           data:
             service-account.json: "{{ gcsServiceAccount | b64encode }}"
+    - name: Create Docker mirror proxy secret
+      k8s:
+        kubeconfig: '{{ automationKubeconfig }}'
+        context: '{{ masterClusterContext }}'
+        state: present
+        namespace: "{{ prowNamespace }}"
+        definition:
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: docker-mirror-proxy
+          type: Opaque
+          data:
+            ca.crt: "{{ dockerMirrorProxyCA.cert }}"
+            ca.key: "{{ dockerMirrorProxyCA.key }}"
```

Comment:
Could you create an extra config-map where you put ca.crt and the publick part of the ca into? Let's keep the key and the ca in an extra secret.

Reply:
Sure done, now the config map embeds the certificate contents and is under templates while no variables are interpolated, let me know if we should use a separate file. I've also updated the secret PR to only include ca.key.